### PR TITLE
Move Daml Helm Chart and Auth0 setup pages to early access

### LINKED ISSUE #13038: Move Helm Chart docs to early access and point out it doesn't work with Daml 2.0

### LINKED ISSUE #13039: Move Auth0 setup docs to early access and point out it doesn't work with Daml 2.0

### PR DESCRIPTION
**Pull request by stefanobaghino-da**
Add a disclaimer that these pages do not work with Daml 2.0
but are under active development.

Closes #13038
Closes #13039

changelog_begin
changelog_end

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [x] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description
- [ ] If you mean to change the status of a component, please make sure you keep [the Component Status page](https://github.com/digital-asset/daml/blob/main/docs/source/support/component-statuses.rst) up to date.

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

**Comment by stefanobaghino-da** [comment]
## Previews

### Index

Note the change in hierarchy on the left side

index

### Helm Chart

Note the disclaimer at the beginning of the page

helm

### Auth0 Setup

Note the disclaimer at the beginning of the page

auth0

**Review by bame-da — APPROVED** [review]
Works for me.

**Review by davidrichards-da — APPROVED** [review]
LGTM

File: docs/source/index.rst [hunk 1]
```diff
@@ -96,6 +96,7 @@ Daml Documentation
    tools/visual
    concepts/interoperability
    tools/non-repudiation
+   ops/connect/index
```
Comment (cocreature) [inline docs/source/index.rst:99]: the reason this is failing is because you also need to update `docs/configs/pdf/index.rst`.

Comment (stefanobaghino-da) [reply docs/source/index.rst:99]: Thanks for the help. I probably missed it. Where was this documented?

Comment (stefanobaghino-da) [reply docs/source/index.rst:99]: Addressed by https://github.com/digital-asset/daml/pull/13052/commits/a147a340ab6771dab7284a3cc6061f6d096ba6d2

Comment (cocreature) [reply docs/source/index.rst:99]: not aware of any docs for that. Note that thanks to our amazing new release process you also need to duplicate those changes in the assembly repo which also has separate html & pdf index files.

**Pull request merged**